style.css

```diff
@@ -41,7 +41,7 @@ html{
 
 #nav-bar ul li{
     display: inline-block;
-    margin-right: 50px;
+    margin-right: 40px;
 }
 
```

Comment:
- [ ] You might be able to add a why here for first time students, you can say something like "Since customers usually want their designs perfectly implemented, no matter how much better it could be"